mopidy/core/library.py

```diff
@@ -170,14 +170,33 @@ def lookup(self, uri):
         them all.
 
         :param uri: track URI
-        :type uri: string
-        :rtype: list of :class:`mopidy.models.Track`
+        :type uri: string or :class:`None`
+        :param uris: track URIs
+        :type uris: list of string or :class:`None`
+        :rtype: list of :class:`mopidy.models.Track` if uri was set or a
+            ``{uri: list of tracks}`` if uris was set.
```

Comment:
Since there's different return structures here, we might as well go all in with core-cleanup in the docs and say `{uri: list of models}`. As in, don't promise tracks.